### PR TITLE
Repo refresh interval setting

### PR DESCRIPTION
**Pull request by pylerSM**

### CODE REVIEW

**Comment by DVDAndroid** [comment]
Have you tested these changes?

File: app/src/main/java/de/robv/android/xposed/installer/util/UpdateService.java [hunk 1]
```diff
@@ -54,7 +54,10 @@ public void onCreate() {
         super.onCreate();
 
         SharedPreferences prefs = getSharedPreferences(getPackageName() + "_preferences", MODE_PRIVATE);
-        int interval = Integer.parseInt(prefs.getString("update_service_interval", "3"));
```
Comment (DVDAndroid) [inline app/src/main/java/de/robv/android/xposed/installer/util/UpdateService.java:57]:
```
Caused by: java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
at android.app.SharedPreferencesImpl.getInt(SharedPreferencesImpl.java:240)
at de.robv.android.xposed.installer.util.UpdateService.onCreate(UpdateService.java:57)
```

File: app/src/main/res/xml/prefs.xml [hunk 2]
```diff
@@ -114,6 +114,14 @@
...
             android:title="@string/open_drawer"/>
 
+        <de.robv.android.xposed.installer.widget.ListPreferenceSummaryFix
+            android:defaultValue="24"
+            android:entries="@array/update_text"
+            android:entryValues="@array/update_values"
+            android:key="repo_refresh_interval"
+            android:summary="%s"
+            android:title="@string/repo_refresh_interval"/>
+
```
Comment (DVDAndroid) [inline app/src/main/res/xml/prefs.xml:125]: Place this preference under other "update_service_interval" preference

**Comment by pylerSM** [comment]
Ehmm. I will fix it tomorrow

**Comment by DVDAndroid** [comment]
Ok no problems

**Comment by DVDAndroid** [comment]
I think that the repo refresh interval is unnecessary because the repos list get updated only when app is alive.

**Pull request closed without merging**